cypress/support/fragments/bulk-edit/bulk-edit-actions.js

```diff
@@ -1,16 +1,49 @@
-import { Button, HTML } from '@interactors/html';
+import { Button, HTML, including } from '@interactors/html';
 import FileManager from '../../utils/fileManager';
-import { Modal } from '../../../../interactors';
+import { Modal, SelectionOption } from '../../../../interactors';
```

Comment:
Should I merge lines 1 and 3 into one?

Reply:
Unfortunatly no because I import interactors from different places
3 line from '../../../../interactors'
and 1 line from '@interactors/html';